frontend/app/model/Schema.scala

```diff
@@ -25,8 +25,17 @@ case class OfferSchema(
  category: String,
  price: String,
  priceCurrency: String,
+ validFrom:String,
```

Comment:
I'd add a space between `validFrom:` and `String` on this line, for readability/formatting consistency.